api/grpcserver/interface.go

```diff
@@ -54,9 +54,10 @@ type postSetupProvider interface {
 	Config() activation.PostConfig
 }
 
-// peerCounter is an api to get amount of connected peers.
-type peerCounter interface {
+// Peers is an api to get peer related info.
+type Peers interface {
```

Comment:
Interfaces are usually private if they only exist to be able to mock a dependency (i.e. are not intended to be used outside the package they are defined in).